src/utils/legacy.ts

```diff
@@ -1,6 +1,7 @@
-/*
+/**
```

Comment:
Its not really a doc block so lately we did not use the second `*` to prevent issues with doc extraction - but :shrug: